.github/actions/custom-actions/aselo_development_custom/action.yml

```diff
@@ -29,6 +29,23 @@ runs:
       #   with:
       #     ssm_parameter: "DEV_AS_IWF_API_COUNTRY_CODE"
       #     env_variable_name: "IWF_API_COUNTRY_CODE"
+
+    - name: Set IWF_SECRET_KEY
```

Comment:
Will the SSM parameters need to be scoped to the Twilio account, at least the secret key, looks like the report URL too? e.g., `DEV_IWF_AS_SECRET_KEY`

Reply:
Is this a new key that will be created? Does it make sense to go ahead and start using the new SSM naming convention to reduce the number of things that need to migrate?

I'm happy to make some suggestions about the structure if so. I'll need to ask some questions about the context of the secret to make those, though.

Reply:
@robert-bo-davis - sorry I missed this - we'll use the legacy convention for this case but start using the new one going forward

Reply:
@stephenhand No worries. Stephen and I had already discussed and come to the same conclusion.